cf_remote/main.py

```diff
@@ -118,8 +118,8 @@ def _get_arg_parser():
     dp.add_argument("name", help="Name fo the group of hosts to destroy", nargs='?')
 
     sp = subp.add_parser("deploy", help="Deploy masterfiles to hub")
-    sp.add_argument("--hub", help="Hub(s) to deploy to", type=str, required=True)
-    sp.add_argument("masterfiles", help="Path to local masterfiles directory or tarball", type=str)
+    sp.add_argument("--hub", help="Hub(s) to deploy to", type=str)
+    sp.add_argument("masterfiles", help="Path to local masterfiles directory or tarball", type=str, nargs="?")
```

Comment:
It can also be a URL, right?

Reply:
Thanks for review: https://github.com/cfengine/cf-remote/pull/30